ci/test_notebooks.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
@bdice, I see this `test_notebooks.sh` file, but I don't see a corresponding job in `pr.yaml`.

Also, it doesn't seem that `cucim` notebooks are currently tested in Jenkins PRs.

Therefore, should we punt notebooks testing for `cucim` for a separate PR to prevent scope creep?

Reply:
I was going to make a quick attempt at running notebooks and defer to a later PR if it does not work.

In general, my strategy for all the tasks we want to do in this release is to try things aggressively, and reduce scope if an “optional” task fails after some small, fixed amount of effort. Some bets pay off, others do not, but I try to keep the downside loss (budget of effort for non-essential goals) small.